micrometer-core/src/main/java/io/micrometer/core/instrument/binder/http/HttpObservationDocumentation.java

```diff
@@ -0,0 +1,297 @@
+ *
+ * @author Brian Clozel
+ * @author Marcin Grzejszczak
+ * @since 1.12.0
+ */
+public enum HttpObservationDocumentation implements ObservationDocumentation {
+
+    /**
+     * Observation created when a request is received with Javax Http.
+     */
+    SERVER_OBSERVATION {
+        @Override
+        public Class<? extends ObservationConvention<? extends Context>> getDefaultConvention() {
+            return DefaultHttpServerRequestObservationConvention.class;
+        }
+
+        @Override
+        public KeyName[] getLowCardinalityKeyNames() {
+            return KeyName.merge(CommonLowCardinalityKeys.values(), ServerLowCardinalityKeys.values());
+        }
+
+        @Override
+        public KeyName[] getHighCardinalityKeyNames() {
+            return CommonHighCardinalityKeys.values();
+        }
+
+    };
+
+    public enum CommonLowCardinalityKeys implements KeyName {
+
+        /**
+         * HTTP request method.
+         */
+        HTTP_REQUEST_METHOD {
+            @Override
+            public String asString() {
+                return "http.request.method";
+            }
+        },
+
+        /**
+         * HTTP response status code.
+         */
+        HTTP_RESPONSE_STATUS_CODE {
+            @Override
+            public boolean isRequired() {
+                return false;
+            }
+
+            @Override
+            public String asString() {
+                return "http.response.status_code";
+            }
+        },
+
+        /**
+         * OSI Application Layer or non-OSI equivalent. The value SHOULD be normalized to
+         * lowercase.
+         */
+        NETWORK_PROTOCOL_NAME {
+            @Override
+            public String asString() {
+                return "network.protocol.name";
+            }
+        },
+
+        /**
+         * Name of the local HTTP server that received the request.
+         */
+        SERVER_ADDRESS {
+            @Override
+            public String asString() {
+                return "server.address";
+            }
+
+        },
+
+        /**
+         * Port of the local HTTP server that received the request.
+         */
+        SERVER_PORT {
+            @Override
+            public String asString() {
+                return "server.port";
+            }
+
+        },
+
+        /**
+         * The URI scheme component identifying the used protocol.
+         */
+        URL_SCHEME {
+            @Override
+            public String asString() {
+                return "url.scheme";
+            }
+        },
+
+        /**
+         * Name of HTTP request method or {@value KeyValue#NONE_VALUE} if the request was
+         * not received properly.
+         */
+        METHOD {
+            @Override
+            public String asString() {
+                return "method";
+            }
+
+        },
+
+        /**
+         * HTTP response raw status code, or {@code "UNKNOWN"} if no response was created.
+         */
+        STATUS {
```

Comment:
There is also a `HTTP_RESPONSE_STATUS_CODE`. Do we need both?

Reply:
Yes, this one is for backward compatibility (our existing instrumentations tag like that). The other one is following semantic conventions.